Add accessibility policy

hobbs-Hobbler opened the pull request:
## What?
Adds a new document containing the Hubs Foundation Accessibility Policy. This policy strives to bind all Hubs Foundation-produced content and all working teams.  The overall theme is for contributors to try their best, but some specific guidance examples (not expressed as rules) is provided.

Clarifies what HF means by accessibility in this policy . It is considered to be the *human ability* definition of vision, sound, mobility, and cognition.  It is not the technology-based definition of computing burden, speed, or ease of use.

Emphasizes community (by hopefully drawing in the 2 threads of open-source and community-based) for accessibility efforts. 

Includes a way of either reporting accessibility failures or at least discussing concerns. Also provides 2 methods to do those: private by email and public by GitHub Issue.

## Why?
As good practice, all organizations are expected to have an accessibility policy.  A policy also helps define expectations and create a common communication platform about issues.


## Limitations
No part of this policy is retroactive.

The policy was not written restrictively only to the Hubs Foundation website, for example.  It is expected to be relevant to all Hubs Foundation content creators, that is, all working teams (Development, Documentation, Art, etc.).

This policy is NOT bound upon Hubs users.

This policy is not bound upon any auxiliary software or platforms that HF uses to conduct it's business when those platforms do or do not not contain accessibility options. For example, it is not bound to HF in it's Blender files, GitHub code, or tax documents beyond what is provided by those media/platforms.

Generally the idea is that when Hubs Foundation directly endorses or publishes content, the content expression is bound by this accessibility policy, to the best of the HF ability within the platform/media constraints.  Here are some working examples:

1. HF publishes an announcement to the HF website. It is bound by this policy because the website is under the **reasonable direct control** of HF. Because accessibility options can be utilized, they should be.
2. HF publishes an announcement to Discord. It is bound only by what accessibility options Discord can provide.
3. HF produces a how-to video. It is bound by this policy and should be as accessible as possible (with an accurate transcript with visual descriptions, if necessary, included).
4. A writer contributes a blog post to HF Creator Labs. The expression of that blog post should be as accessible as possible, being published under the direct editing of the Hubs Foundation. Thus, images should have alt text and document formatting should be used.  If the writer co-publishes the same post to a non-HF controlled location, that written expression is not bound by this policy.
5. If a contributor makes a how-to video and shows it to the Hubs community, it is not, at least initially bound to this policy. However, HF can request that, in order to increase access for our community, the contributor provide a transcript with the video. Then both the video and the transcript could be shared in good faith by HF (for example, as a Community Resource) as meeting the spirit of this policy.

As an XR product, we must comment on the accessibility options (or lack thereof) in the XR community in general.  While it is not expected that Hubs will lead the way in innovative accessibility options (for example, text-to-speech for 3D objects that appear in Hubs), Hubs Foundation acts as a global citizen.  I expect that Hubs will have that text-to-speech someday. However, at this moment, this policy binds HF to strive for more accessibility **_when possible_**.  It means that when we can help our fellow citizens, we will.


## Alternatives considered
Some accessibility policies are very short (e.g. [IEEE's](https://www.ieee.org/accessibility-statement)) but they appear restricted to just websites.
Directing users to checklists or long, confusing documents (e.g. some [Web3C materials](https://www.w3.org/TR/2021/NOTE-xaur-20210825/) are very long) does not meet the spirit of a sharing and collaborative community--it can cause users to give up in frustration.


## Open questions
Included an optional footer for consideration. I'm not sure what I think of it. It's good "hey this is only a draft" language that seems to last much longer than a draft normally could/would.  I didn't notice when I included it that the page (at W3C) seems to point to non-draft equivalent pages (?I don't know how I got to a draft page then?).  The wording is only included here for general consideration to HF; it is not a must.


## Additional details or related context
Sources and included links run the gamut between opinion pieces (while strong, are essentially _opinions_) and hyper-bureaucratized university pages.  The end feeling of perusing these extremes is that "just try your best" is good working advice.  Still, I included a few tips that I've used repeatedly in my writing for HF.

hobbs-Hobbler committed:
Add accessibility policy

What: Adds a new document containing the Hubs Foundation
guidance and stance on accessibility.

Why: To guide Hubs Foundation contributors in the making
of accessible content. Defines accessibility with relation
to the Hubs Foundation mission and Hubs software.

Note: first draft

hobbs-Hobbler commented:
Oops, I'll need help removing 6 unneeded commits. (eye roll!)

hobbs-Hobbler commented:
BTW, I used no AI, but I did use a thesaurus.

DougReeder commented:
This looks reasonable to me

hobbs-Hobbler commented:
Force push from December 15 was to drop unneeded commits.  My thanks to @Exairnous for the guidance and instructions.

hobbs-Hobbler committed:
Adds technical writing guidelines

What: Transfers writing guidelines from the Website
Style Guide to the Accessibility Policy.
Includes guidance on action verb use, bolding, italics,
list numbering, menus,capitalization, links,
and discouraged language choices. Examples included.
Three new images added.

Why: The Website Style Guide's existence is not widely
known outside of the Communication and Documentation Team.
This transfers guidance from that Google document to
our GitHub repository.

Typically, reviewing for accessibility is a late-stage edit.
This puts writing-for-accessibility at an earlier stage by
making these tips easier to find and reference.

Note: Most of these entries were reviewed, modified, and approved
at the November 26, 2025 Documentation Team Meeting. Slight changes have
been made to clarify text and examples. In a few cases, the
writing discouraging bolding and italics has been moderated to
more freely allow their use, due to subsequent screen reader testing.

There was also some discussion of clarity in link text at Discord here:
https://discord.com/channels/498741086295031808/1258349074990759986/1446075871743180863
where it was agreed that the text for links can be very specific as to
*where* the link is going (e.g. Discord, GitHub, etc.). This commit
reflects that agreement.

hobbs-Hobbler committed:
Fixes images markdown

What: Modifies markdown for images. I had added "img/" to the
front of the link thinking it was necessary in Markdown.  Testing
shows me it is not.

Why: Helps images to be visible within the policy.

Note: extremely minor fix and I figured it out myself. Yay me.

hobbs-Hobbler commented:
Insert confused face. I have the images correctly working in Preview mode in VS Code, but I don't see them working here in GitHub. Hmm...  I'll have to solve this on a future day.

Exairnous commented:
@hobbs-Hobbler 
> Insert confused face. I have the images correctly working in Preview mode in VS Code, but I don't see them working here in GitHub. Hmm... I'll have to solve this on a future day.

I think it might be that images just don't show in the pull request rich diff (I checked your PR for the moderator guidelines, because that document has an image that I know works, and it's image didn't render either in the PR).  That said, I notice that you haven't included a beginning slash in your image paths, which may or may not be required (I'm not sure, but the image links in the other documents have it - though changing it still won't make the image render in the PR).  See the link below for reference.

https://github.com/Hubs-Foundation/policies-procedures-guidelines-public/blob/main/showcasing-community-events.md?plain=1#L20

hobbs-Hobbler committed:
Adds slash to beginning of image path addresses

What: Adds a slash to the beginning of the 3 previously
comitted images.

Why: Typo and probably not having the slash was the incorrect
path, although some testing didn't help clarify this. Addresses
commit comment on the path topic.

Note: when I went to add in the slash in front of the image paths,
VS Code helpfully suggested the "images" folder and when I selected that,
VS Code suggested the images inside that folder.  So putting the slash
in at the front was/is probably the solution. My thanks for the
commit comments.

hobbs-Hobbler commented:
@Exairnous I addressed your comment. Thank you!  This is ready for another review.

hobbs-Hobbler commented:
FWIW, even though I was *sure* I had copies of the images in BOTH policy repositories on my computer because I double checked (don't ask about having more than 1 of the same repository, it's related to starting all over a new repository each time I sat down to a project), when I went back and looked at the *correct* repository yesterday (b/c I remembered that I could look at my code path in the VS Code Terminal window), and I noticed that the 3 new images...were not there.  Sigh, bother. Oh well!! I'm glad to think that basically everything went OK or is going OK on the GitHub end.
